features/events/traps/src/main/resources/OSGI-INF/blueprint/blueprint-trapd-listener.xml

```diff
@@ -52,17 +45,6 @@
 	</bean>
 	<service ref="trapdConfig" interface="org.opennms.netmgt.config.TrapdConfig" />
 
-	<!-- The TrapdConfiguration is retreived via the OpenNMS ReST API every n ms and merged with the trapdConfiguration by invoking "trapdConfig.onUpdate"-->
-	<!-- @see http://www.davsclaus.com/2012/06/locking-down-jmx-naming-now-easier-with.html -->
-	<camelContext id="trapdListenerContext" managementNamePattern="#name#" xmlns="http://camel.apache.org/schema/blueprint">
-		<route id="restClientToTrapdConfig">
-			<from uri="timer://restTrapConfigurationTimer?fixedRate=true&amp;period={{trapd.config.interval}}"/>
-			<bean ref="restClient" method="getSnmpV3Users"/>
-			<bean ref="unmarshaller" />
-			<bean ref="trapListener" method="setSnmpV3Users" />
-		</route>
-	</camelContext>
-
 	<!--
 		The TrapListener handles received traps (the ones we were listening for) and forwards
 	 	them via the MessageProducer (obtained by the MessageProducerFactory)
```

Comment:
Need to get TwinSubscriber reference and add it as property in trapListener.
Also add Twin API bundle in features.